source/test/TestApp/TestRunner.cpp

```diff
@@ -105,8 +105,8 @@ namespace PlayFabUnit
     {
         std::stringstream summaryStream;
 
-        PlayFab::TimePoint timeNow = PlayFab::GetTimePointNow();
-        PlayFab::TimePoint testStartTime, testEndTime;
+        Int64 timeNow = PlayFab::GetMilliTicks();
+        Int64 testStartTime, testEndTime;
```

Comment:
It seems to me that testStartTime and testEndTime are only ever written to and never read.  Is there a logic problem here?